docs/api-composite.md

```diff
@@ -28,6 +28,7 @@ If the overlay image contains an alpha channel then composition with premultipli
         -   `options.create.height` **[Number][4]?** 
         -   `options.create.channels` **[Number][4]?** 3-4
         -   `options.create.background` **([String][2] \| [Object][3])?** parsed by the [color][6] module to extract values for red, green, blue and alpha.
+    -   `options.blendMode` **[String][2]** [blend mode][8] to composite images with (optional, default `'over'`). Possible values: `over`, `darken`, `multiply`, `color burn`, `lighten`, `screen`, `color dodge`, `overlay`, `soft light`,  `hard light`, `difference`, `exclusion`
```

Comment:
The `docs/api-*.md` files are auto-generated from JSDocs in the source. Please can you add this content to the JSDocs for the `overlayWith()` method then use `npm run docs` to refresh the markdown file.